baseapp-auth/testproject/settings.py

```
@@ -21,12 +26,17 @@
     "graphene_django",
 ]
 
+MIDDLEWARE = [
+    *MIDDLEWARE,
```

Ruff

`MIDDLEWARE` may be undefined, or defined from star imports

(F405)